examples/python-betterproto/test_betterproto.py

```
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
+"""Test betterproto's modern Python approach."""
+
+import asyncio
+import sys
+sys.path.insert(0, '.')
+
+# Import will be from betterproto generated code
+# The import path may differ from standard protobuf
+try:
+    from proto.gen.python.modern import Product, ProductService, CreateProductRequest
```

Ruff (0.11.9)

`proto.gen.python.modern.ProductService` imported but unused; consider using `importlib.util.find_spec` to test for availability

(F401)